session.go

```diff
@@ -429,38 +447,58 @@ func (s *Session) sendLoop() error {
 		default:
 		}
 
+		var buf []byte
+		// Preferentially use control channel.
+		select {
+		case buf = <-s.sendCtrlCh:
+			goto WITHDATA
```

Comment:
nits:

* I'd just call this `send`.
* Labels usually aren't uppercased.

Reply:
Labels are uppercased in their other uses in this and stream.go.

Reply:
Ah, well, I didn't write that code either. Nevermind, consistency is king.